doc/nrf/releases/release-notes-1.9.0.rst

```diff
@@ -0,0 +1,840 @@
+      * Documentation by adding information about LTE fallback mode. See the section Connection fallback mode in the library documentation.
+
+    * Fixed:
+
+      * An issue where subsequent calls to :c:func:`lte_lc_connect` or :c:func:`lte_lc_init_and_connect` could hang until timeout, and in some cases change the LTE mode.
+
+* Removed:
+
+  * AT command interface library.
+  * AT command notifications library.
+
+Zigbee libraries
+----------------
+
+* Updated:
+
+  * :ref:`lib_zigbee_osif` library documentation by updating the Configuration section with new Kconfig options.
+  * :ref:`lib_zigbee_error_handler` library documentation with information about the new function :c:func:`zb_osif_abort`.
+
+Other libraries
+---------------
+
+* Added:
+
+  * New library :ref:`mod_dm`.
+
+* Updated:
+
+  * :ref:`lib_bootloader` library:
+
+    * Moved :ref:`lib_bootloader` to a section of their own.
+    * Added write protection by default for the image partition.
+
+  * :ref:`lib_date_time` library:
+
+    * Removed the :kconfig:`CONFIG_DATE_TIME_IPV6` Kconfig option.
+      The library now automatically uses IPv6 for NTP when available.
+    * Updated the documentation as part of refactoring the NTP server selection logic.
+
+  * :ref:`lib_location` library:
+
+    * Added support for GNSS high accuracy.
+
+  * :ref:`lib_ram_pwrdn` library:
+
+    * Added:
+
+      * Functions for powering up and down RAM sections for a given address range.
+      * Experimental functionality to automatically power up and down RAM sections based on the libc heap usage.
+      * Support for nRF5340 application core to power up and power down RAM sections.
+
+  * :ref:`event_manager`:
+
+    * Added:
+
+      * ``EVENT_SUBSCRIBE_FIRST`` subscriber priority and updated the documentation about this functionality.
+
+    * Updated:
+
+      * The sections used by the event manager and stopped using orphaned sections.
+
+        * Event manager no longer uses orphaned sections to store information about event types, listeners, and subscribers.    Hence, the :kconfig:`CONFIG_LINKER_ORPHAN_SECTION_PLACE` option is no longer required in the config file.
+
+      * Reworked priorities.
+
+
+    * Removed:
+
+      *  Forced alignment for x86.
+
+  * :ref:`event_manager_profiler_tracer`:
+
+    * Updated:
+
+      * Event manager Profiler Tracer to no longer use orphaned sections to store profiler information.
+        Hence, the :kconfig:`CONFIG_LINKER_ORPHAN_SECTION_PLACE` option is no longer required in the config file.
+
+TF-M libraries
+--------------
+
+* Updated:
+
+  * :ref:`lib_tfm_ioctl_api`:
+
+    * Updated the documentation about the IOCTL request types that have been moved to the upstream TF-M platform support.
+
+
+sdk-nrfxlib
+-----------
+
+See the changelog for each library in the :doc:`nfxlib documentation <nrfxlib:README>` for additional information.
+
+* Added:
+
```

Comment:
Referencing point in highlights on nrf_security additions:

Maybe add a point here on adding PSA crypto drivers for CryptoCell and Oberon instead of having that part in the release highlights

Reply:
Added